src/modules/module_3/eventEmitterClass.js

```diff
@@ -0,0 +1,59 @@
+console.log('event Emitter Class')
+
+const EventEmitter = require("events");
+
+class MyEventEmitter extends EventEmitter {
```

Comment:
Task-1 from 'Module 3' specifies writing a copy of EventEmitter, not just wrapping it.
Will have to deduct 10 points. Need an update here.